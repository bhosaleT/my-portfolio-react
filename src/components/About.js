import React from "react";

const About = () => (
  <div className="about">
    <img
      className="profile-image"
      src={require("../images/me.png")}
      alt="Tejas-Bhosale"
    />
    <div className="about__text">
      <p className="about-text">
        <span className="highlight"> I'm</span> a newly graduated web developer offering enthusiasm
        and understanding of Front-end frameworks like React, Redux and advanced
        CSS techniques and back-end frameworks like Node.js and mongoDB. I will
        bring to the table the ability to design and develop beautiful and
        functional websites using the latest technologies and web guidelines.
      </p>

      <a className="button" href={require("../resume/resume.pdf")} download>
        <img
          className="button-download"
          src={require("../images/svg/install.svg")}
          alt="download button"
        />
        DOWNLOAD CV{" "}
      </a>
    </div>
  </div>
);

export default About;
