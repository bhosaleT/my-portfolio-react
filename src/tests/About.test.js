import ReactShallowRenderer from "react-test-renderer/shallow";
import React from "react";
import About from "../components/About";

test("Should render About component correctly", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<About />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
