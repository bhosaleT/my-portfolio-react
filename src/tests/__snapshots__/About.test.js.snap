// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render About component correctly 1`] = `
<div
  className="about"
>
  <img
    alt="Tejas-Bhosale"
    className="profile-image"
    src="me.png"
  />
  <div
    className="about__text"
  >
    <p
      className="about-text"
    >
      <span
        className="highlight"
      >
         I'm
      </span>
       a newly graduated web developer offering enthusiasm and understanding of Front-end frameworks like React, Redux and advanced CSS techniques and back-end frameworks like Node.js and mongoDB. I will bring to the table the ability to design and develop beautiful and functional websites using the latest technologies and web guidelines.
    </p>
    <a
      className="button"
      download={true}
      href="resume.pdf"
    >
      <img
        alt="download button"
        className="button-download"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "install.svg",
          }
        }
      />
      DOWNLOAD CV
       
    </a>
  </div>
</div>
`;
